async_upnp_client/dlna.py

```diff
@@ -271,76 +225,84 @@ def _supports(self, var_name: str) -> bool:
 
     def _level(self, var_name: str) -> Optional[float]:
         state_var = self._state_variable('RC', var_name)
-        value = state_var.value
+        if state_var is None:
+            return None
+
+        value = state_var.value  # type: float
         if value is None:
             _LOGGER.debug('Got no value for %s', var_name)
             return None
 
-        max_value = state_var.max_value or 100
+        max_value = state_var.max_value or 100.0
         return min(value / max_value, 1.0)
 
-    async def _async_set_level(self, var_name: str, level: float, **kwargs: Dict[str, Any]) -> None:
+    async def _async_set_level(self, var_name: str, level: float, **kwargs) -> None:
         action = self._action('RC', 'Set%s' % var_name)
-        argument = action.argument('Desired%s' % var_name)
+        if not action:
+            return
+
+        name = 'Desired%s' % var_name
+        argument = action.argument(name)
         state_variable = argument.related_state_variable
+
         min_ = state_variable.min_value or 0
         max_ = state_variable.max_value or 100
         desired_level = int(min_ + level * (max_ - min_))
-        args = kwargs.copy()
-        args.update({'Desired%s' % var_name: desired_level})
 
+        args = kwargs.copy()
+        args[name] = desired_level
         await action.async_call(InstanceID=0, **args)
 
 # region RC/Picture
     @property
-    def has_brightness_level(self) -> bool:
+    def has_brightness_level(self):
```

Comment:
Hm, the return type hint was _removed_ here (and in bunch of other things below), was that on purpose?